src/compas/files/off.py

```diff
@@ -113,7 +87,7 @@ def pre(self):
     def post(self):
         pass
 
-    def read(self):
+    def read_off(self):
```

Comment:
@gonzalocasas:
> I'm not very convinced about calling these `read_off()`, `read_obj()` and so on. I think it should be the same name across the board. I see that now `read()` is used, so, either rename this level of reads to `read_content` or `read_file`, or rename the `BaseReader.read` to `BaseReader.read_bytes` (or `BaseReader.read_line` or `BaseReader.read_chunk`)

Reply:
in general i think it is confusing that `read` does something else then what the corresponding method of a standard file object does 

https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects

Reply:
`parse()`? It's in some of the modules but not used, only as skeleton function.

Reply:
https://github.com/compas-dev/compas/pull/418/files/0682dcf5876f08b75e93f31d36a8e50292a30a8b#r355006802